weave/flow/dataset.py

```diff
@@ -0,0 +1,12 @@
+import typing
+import weave
+
+from .obj import Object
```

Comment:
here the import is relative, elsewhere it is `from weave.flow.obj import Object` would be good to be consistent

Reply:
fixed.